envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/envoy/EnvoyContainer.kt

```diff
@@ -18,7 +18,7 @@ class EnvoyContainer(
     private val certificateChain: String = "/app/fullchain_echo.pem",
     private val privateKey: String = "/app/privkey.pem",
     private val logLevel: String = "info",
```

Comment:
Looks like those are unused.

Reply:
That's true, containers, in general, do require some cleanup. I didn't want to refactor the container code in this PR though, I view it as a separate task to touch.